pkg/tenant/api/v1/client.go

```diff
@@ -100,9 +100,9 @@ func (s *APIv1) SignUp(ctx context.Context, in *ContactInfo) (err error) {
 	return nil
 }
 
-func (s *APIv1) TenantList(ctx context.Context, in PageQuery) (out *TenantPage, err error) {
-	// Set values for PageSize and NextPageToken keys in the PageQuery struct
-	in = PageQuery{2, "12"}
+func (s *APIv1) TenantList(ctx context.Context, size uint32, tok string, in PageQuery) (out *TenantPage, err error) {
+	// Set values for PageSize and NextPageToken in PageQuery
+	in = PageQuery{size, tok}
```

Comment:
Here's a link to the query package to I used to set the values for PageSize and NextPageToken: https://pkg.go.dev/github.com/google/go-querystring@v1.1.0/query#Values

Reply:
I think that this package is a good choice for binding struct values into query params! 

There does seem to be a bit of duplication here; you're having the user pass in `in PageQuery` but then you're overwriting what the user passes in with the `size` and `tok` elements. Wouldn't it be better to just have the user pass in the PageQuery without constructing it again, e.g.

```
func (s *APIv1) TenantList(ctx context.Context, in PageQuery) (out *TenantPage, err error) {
```

Then, if we change the PageQuery parameter (e.g. if we add `PrevPageToken` for backwards pagination) then the API method signatures would not have to change.

Reply:
I looked at the bff pkg and figured out why I was having trouble with this. I've made the updates and will push shortly.